.github/workflows/release.yml

```diff
@@ -11,26 +11,16 @@ on:
       commit:
         description: 'A full-length commit SHA-1 hash'
         required: true
-      skip-docker:
-        description: 'Skip Docker image publishing'
-        type: boolean
-        default: false
-      skip-s3:
-        description: 'Skip S3 artifact publishing'
-        type: boolean
-        default: false
-      skip-version-info:
-        description: 'Skip publishing version-info.json'
-        type: boolean
-        default: false
-      skip-tag:
-        description: 'Skip tagging the release'
-        type: boolean
-        default: false
-      skip-release-notes:
-        description: 'Skip publishing release notes'
-        type: boolean
-        default: false
+      release-channel:
+        description: 'The release channel to publish to (optional)'
+        type: choice
+        options:
+          - none
+          - latest
+          - beta
+          - nightly
+        required: true
+        default: 'none'
```

Comment:
release noob question: in what situations do we set release-channel to 'none'? Just for testing workflows?

Reply:
possibly found the answer in the other PR

https://github.com/metabase/metabase/pull/49093#discussion_r1828170049

Reply:
some situations where we might want to use the "none" release channel

1) patches or minor releases for older versions
2) minor releases for new versions that we may want to let more slowly percolate into the wild before setting them to "stable"